src/components/builds/CreateBuildDialog.tsx

```diff
@@ -118,6 +118,7 @@ export default function CreateBuildDialog(props: Props) {
           editorProps={{ $blockScrolling: true }}
           highlightActiveLine={true}
           showGutter={true}
+          setOptions={{ useWorker: true }}
```

Comment:
PR title is "Disable Ace worker" but here it seems it's getting enabled. What is the right thing?

Reply:
Good catch: I was trying to reproduce the issue to make sure that I get the same error as we see in Sentry, but forgot to revert the change.